pkg/validators/validate_test.go

```diff
@@ -24,19 +24,24 @@ func TestLegacyAPIKeys(t *testing.T) {
 
 func TestPublishableAPIKey(t *testing.T) {
 	err := APIKey("pk_test_12345")
-	require.EqualError(t, err, "the CLI only supports using a secret or restricted key")
+	require.EqualError(t, err, "for security reasons, the Stripe CLI does not permit the use of secret keys in live mode. To generate restricted keys for use in live mode, use the `stripe login` command")
```

Comment:
this error message should be more like "...only supports using a restricted key"

Reply:
Fixed in the latest commit by restructuring the error message logic.

Now, users should see less disruption in their expected error messages, since we only display the new error message when a livemode secret key is passed. In all other cases, the default error message stays the same:`"the CLI only supports using a secret or restricted key"`.